docs/adoc/releasenotes/ReleaseNotes_11_0.adoc

```diff
@@ -210,6 +210,11 @@ Each breadcrumb item is clickable. The breadcrumb can automatically shrink if th
 
 image::{rnimgsdir}/new_widget_breadcrumb_bar_field.png[]
 
+=== Chart
```

Comment:
Please add slightly more information (e.g. various chart types) and an image. This is your chance to promote your widget ;)